Schedule/View/ContentView.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  ContentView.swift
+//
+//  Created by alex_tr on 27.12.2024.
+//
+
+import SwiftUI
+import OpenAPIRuntime
+import OpenAPIURLSession
+
+struct ContentView: View {
+    
+    @ObservedObject var viewModel = ScheduleViewModel()
```

Comment:
Модель передается снаружи в данном случае, потому тут ее инитть не стоит
```suggestion
    @ObservedObject var viewModel: ScheduleViewModel
```

Reply:
Согласен, серьезный косяк, поправил.